delivery_auto_refresh/models/sale_order.py

```diff
@@ -33,8 +34,12 @@ def write(self, vals):
         res = super(SaleOrder, self).write(vals)
         for order in self:
             delivery_line = order.order_line.filtered('is_delivery')
+            digits = self.env.user.company_id.currency_id.decimal_places
```

Comment:
No, you have to use decimal defined on the discount field: dp.get_precision('Discount')

Reply:
Done, thanks!